data/sql/updates/pending_db_world/rev_1681369813867048100.sql

```diff
@@ -0,0 +1,46 @@
+-- ID 21861 (Vision of the Raven God), also set to not specified
+UPDATE `creature_template` SET `type` = 10 WHERE `entry` = 21861;
+
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 21861;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21861, 0, 0, 1, 38, 0, 100, 0, 1, 1, 0, 0, 0, 11, 39426, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - On Data Set 1 - Cast credit'),
+(21861, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - Link - Say 0'),
+(21861, 0, 2, 3, 38, 0, 100, 0, 1, 2, 0, 0, 0, 11, 39428, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - On Data Set 2 - Cast credit'),
+(21861, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - Link - Say 1'),
+(21861, 0, 4, 5, 38, 0, 100, 0, 1, 3, 0, 0, 0, 11, 39430, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - On Data Set 3 - Cast credit'),
+(21861, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 2, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - Link - Say 2'),
+(21861, 0, 6, 7, 38, 0, 100, 0, 1, 4, 0, 0, 0, 11, 39431, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - On Data Set 4 - Cast credit'),
+(21861, 0, 7, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 3, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Vision of Raven God - Link - Say 3');
+
+-- Remove cast time
+UPDATE `gameobject_template` SET `Data0` = 0 WHERE `entry` IN (184950,184967,184968,184969);
+
```

Comment:
Trinity uses 56 for Data0, and Data fields are sniffed for non-server gobs

Reply:
I know, but having any lock.dbc value causes them to have a short opening time

![image](https://user-images.githubusercontent.com/46330494/233027891-ea8f3d9d-d158-4839-bb48-d57ff489b4de.png)

Unless they had opening time in OG wotlk and it's instant in recycled?

Reply:
Ohhhhhhhhh, i think the invisible trigger creatures work on proximity and instantly trigger the RP when you get close.